tmt/steps/execute/internal.py

```diff
@@ -189,7 +189,7 @@ def log(
         # Execute the test, save the output and return code
         start = time.time()
         try:
-            stdout, stderr = guest.execute(
+            stdout, _ = guest.execute(
```

Comment:
The `Guest.execute()` method has been modified to always return the named tuple `CommandOutput`. So what about using `stdout = guest.execute(...).stdout` instead?

Reply:
Ok, got your point!